internalsdk/pro/pro.go

```diff
@@ -382,153 +352,9 @@ func (c *proClient) UserLinkValidate(ctx context.Context, code string) (*UserRec
 // PurchaseRequest is used to request a purchase of a Pro plan is will be used for all most all the payment providers
 func (c *proClient) PurchaseRequest(ctx context.Context, data map[string]interface{}) (*PurchaseResponse, error) {
 	var resp PurchaseResponse
-	err := c.webclient.PostJSONReadingJSON(ctx, "/purchase", data, nil, &resp, c.defaultHeader())
-	if err != nil {
-		return nil, err
-	}
-	return &resp, nil
-}
-
-// Auth APIS
-// GetSalt is used to get the salt for a given email address
-func (c *proClient) GetSalt(ctx context.Context, email string) (*protos.GetSaltResponse, error) {
-	var resp protos.GetSaltResponse
-	err := c.webclient.GetPROTOC(ctx, "/users/salt", map[string]interface{}{
-		"email": email,
-	}, &resp, nil)
-	if err != nil {
-		return nil, err
-	}
-	return &resp, nil
-}
-
-// Sign up API
-// SignUp is used to sign up a new user with the SignupRequest
-func (c *proClient) SignUp(ctx context.Context, signupData *protos.SignupRequest) (bool, error) {
-	var resp protos.EmptyResponse
-	err := c.webclient.PostPROTOC(ctx, "/users/signup", nil, signupData, &resp, c.defaultHeader())
```

Comment:
Did we need to use the default headers for the `/users/signup` endpoint or was that a typo?